guidelines/index.html

```diff
@@ -3755,8 +3755,14 @@ <h2 id="x6-glossary"><span class="secno">6. </span>用語集<span class="permali
    <p>共通の目的を共有し、同じコンテンツ制作者、グループ、又は組織により制作された<a href="#dfn-web-page-s" class="internalDFN" data-link-type="dfn">ウェブページ</a>の集合。
    </p>
    
-   <p class="example">例には複数のウェブページにまたがる発行物が含まれ、各ページには 一つの章又はその他の重要なセクションが含まれる。論理的に発行物は 一つの連続した単位であり、ウェブページ一式へのアクセスを可能にするナビゲーション機能を有する。
-   </p>
+   <aside class="example" id="example-26"><div class="marker">
+    <a class="self-link" href="#example-26">例<bdi> 26</bdi></a>
+  </div><p>以下の例がある:</p>
+      <ul>
+      <li>複数のウェブページにまたがる発行物で、各ページには一つの章又はその他の重要なセクションが含まれる。論理的に発行物は一つの連続した単位であり、ウェブページ一式へのアクセスを可能にするナビゲーション機能を有する。</li>
```

Comment:
ここのunitは「単位」ではないような。カタカナで「ユニット」としてよいニュアンスだと思います (アイドルユニットとかユニットバスとかのユニット)。日本語にするなら「集合体」というくらいでしょうか?

最後の文では "the full set of pages." を「ウェブページ一式」としてしまっていますが、"set of web pages" とは別の言葉ですから別の訳し方をした方が良いと思います。

```suggestion
      <li>複数のウェブページにまたがる発行物で、各ページには一つの章又はその他の重要なセクションが含まれる。その発行物は論理的には一つの連続した集合体であり、ページ一式全体へのアクセスを可能にするナビゲーション機能を有する。</li>
```